### DISCUSSION TITLE
Request features you would like to see

### DISCUSSION THREAD

**Discussion by Denellyne** [Denellyne/PCXSense#1] [General]
Im currently working on the GUI but im open for any ideas

**Comment by DrLegitamate** [comment]
Hello, I think i know the answer already but it cant hurt to ask. Is this only for windows? and if it is would Linux ever be considered?

**Reply by Denellyne** [reply]
Yes it's for windows only,somethings I implement are using the winapi and given I'm not familiar with linux then its a bit hard to port ,that and I would need to maintain 2 versions of the app and Im only one person and i study + work ahaha sry but hope it answers

**Reply by DrLegitamate** [reply]
Yes it does thank you. Either way well done on the great work! Cant wait to see where the project goes

**Reply by Denellyne** [reply]
Thank you haha

**Comment by rx783as** [comment]
Hello, one request here: would you help to add DualSense Edge support?

**Reply by Denellyne** [reply]
I would love to and I actually think I can do it easily,its like 4 lines of code , I just need someone to test it for me given I don't own one

**Comment by Trihy** [comment]
Turbo buttons will be nice to see too. But I have no idea how can this be implemented.

Also profile swap with controller will be good (ds4 does this with two finger swipe on touchpad)

**Reply by Denellyne** [reply]
Turbo buttons are easy I think, i dont understand your second statement though

**Reply by Trihy** [reply]
Hi. For having controller profiles change on the fly, while gaming, or doing other stuff, without having to go to pcxsense ui. You could change profiles for ie, doing 2 finger swipe gesture on touchpad or any other combo. I think touchpad isnt implemented yet, but could be handy to have a way to change proiles without going to pcxsense ui.

**Reply by Denellyne** [reply]
Aight !!

**Reply by Trihy** [reply]
This one isnt implemented yet, right?

Also would be nice to have a profile that completely disables pcxsense, disabling xinput emulation too. So you can select that profile when running a game with native ds5 support. So you dont need to close pcxsense.

**Comment by MrRad0m1zeR** [comment]
Would it be possible to to have a way to stop rumble on the UI it self and not game settings. If it possbile then please this could be a good way to save battery if I am playing on bluetooth.

**Reply by Denellyne** [reply]
You mean you want an option to disable rumble is that it?

**Reply by Trihy** [reply]
What about also adding rumble on button press? I mean you can select which button will rumble when you press it. That could easily add rumble to old emulator games. At maximum rumble level will be hard to tolerate, but at lower levels or selectable will be nice.

**Reply by Denellyne** [reply]
I can make a toggle that when enabled makes that each button adds a bit of rumble when pressed ,you can select how much and each motor you want,Is that ok?

**Reply by Trihy** [reply]
That would be great. Not sure how difficult can be make it for each button separately. Maybe you want rumble on punch button but not on jump button. Still will be great.

**Comment by PFLI2** [comment]
Duakshock 3 support.

**Reply by eVenent** [reply]
Agree, after an unclear future of DS4Windows it would be nice to have a serious alternative. DS4W even introduced accelerometer support, but never let remap pressure-sensitive face buttons to analog triggers or analog sticks. ReWASD has support for face buttons but not for accelerometer. ☹️ DSX could let use DS3 as DS4/DS what would help with games support, because they are recognizing DS3 as DirectInput controller. After proper support for the new Playstation's controller, DSX can add support for old controllers like DS3, DS2 and DS1. One app for all Playstation's fans. ❤️

**Reply by Denellyne** [reply]
Yeah but i don't own a DS3 controller neither know anyone who does, I only got DS4 support because a friend let me borrow his controller

**Reply by Trihy** [reply]
DS3 needs his own driver I think. I can test if you need. BTW, is multiplayer support available yet? I mean more than one controller?

**Reply by Denellyne** [reply]
Not at the moment no

**Comment by PFLI2** [comment]
> Yeah but i don't own a DS3 controller neither know anyone who does, I only got DS4 support because a friend let me borrow his controller

Happy to pitch in and get you one if it means improving the usability state of DS3 on Windows.

**Reply by Denellyne** [reply]
Shipping to portugal will cost ya a pretty penny,also Im not that confortamble receiving money from someone I don't know because I'm afraid of my privacy,though I could create a Buy me a coffee ig?

**Reply by Trihy** [reply]
There is a nefarious tool that convert ds3 to xinput system wide. No app needed. Can't remember if works wireless too, ds3 uses a custom BT protocol.

But you could just add the vid&pid of ds3 for basic support, profiles, mapping, etc after installing nefarious driver. That driver it's only for ds3, won't affect other devices.

**Reply by eVenent** [reply]
> But you could just add the vid&pid of ds3 for basic support, profiles, mapping, etc after installing nefarious driver. That driver it's only for ds3, won't affect other devices.

Sounds like a great idea!

**Reply by Denellyne** [reply]
Ill need to research that a bit better.

**Comment by ghost** [comment]
Unless I just missed the option, I'd like a way to hide the program into the tray.

**Reply by ghost** [reply]
Also unrelated, but the rumble seems to be a lot stronger compared to DS4Windows/Native. Not sure if this intentional but a slider to reduce the strength would be nice. (Tested with Dualsense Edge on Jedi Survivor)

**Reply by Denellyne** [reply]
Ill check it out.

**Comment by LuisGagoGimenez** [comment]
Isn`t it possible to map l2/r2? Id like to invert l1 l2 and r1 r2.

**Reply by Denellyne** [reply]
Wdym?

**Comment by Hayashii884** [comment]
sorry if this sounds silly but is there a way to make it so I can enable or disable the XInput emulation? I think being able to use custom lightbars are cool especially since Steam only has one mode while yours can have different colors for different battery levels, some games dont play well with the emulation on so I have to turn off the program just for select few games but I just really like the lightbar function so much

**Reply by Denellyne** [reply]
That sounds cool :)
